### PR TITLE
add new tag to create struct and update makefile

### PR DESCRIPTION
**Pull request by haifeiWu**
add new tag to create struct and update makefile

### CODE REVIEW

File: bases/base_mysql.go [hunk 1]
```diff
@@ -101,6 +101,10 @@ func generateMysqlTypes(obj map[string]map[string]string, depth int, jsonAnnotat
 			annotations = append(annotations, fmt.Sprintf("json:\"%s\"", key))
 		}
 
+		if dbAnnotation {
+			annotations = append(annotations, fmt.Sprintf("db:\"%s\"", key))
```
Comment (hantmac) [inline bases/base_mysql.go:105]: What is the function of `dbAnnotation` tag ?

Comment (haifeiWu) [reply bases/base_mysql.go:105]:
```suggestion
			annotations = append(annotations, fmt.Sprintf("db:\"%s\"", key))
```
new sql orm tag in our framework，here https://github.com/huandu/go-sqlbuilder

Comment (hantmac) [reply bases/base_mysql.go:105]: I got it. But, you should add the corresponding parameters to fuckdb.json and give it a default value (false).

Comment (haifeiWu) [reply bases/base_mysql.go:105]: ok thanks, i will fix it

**Review by hantmac — APPROVED** [review]

**Review by hantmac — APPROVED** [review]

**Pull request merged**